README.md

```diff
@@ -135,132 +135,132 @@ A `Bot` is a Wechaty instance that control a specific [wechaty-puppet](https://g
 
 ### 4 Class `Room`
 
 All wechat rooms(groups) will be encapsulated as a Room.
 
 | Room | API | Description |
 | :--- | :--- | :---        |
-| static | `create(contactList: Contact[], topic?: string): Promise<Room>` | create a new room |
-| static | `find(query: string): Promise<null \| Room>` | Try to find a room by filter. If get many, return the first one. |
-| static | `findAll(query: string): Promise<Room[]>` | Find all contacts in a room |
-| static | `load(query: string): Room` | load room by room id |
+| static | [`create(contactList: Contact[], topic?: string): Promise<Room>`](https://chatie.io/wechaty/#Room.create) | create a new room |
+| static | [`find(query: string): Promise<null \| Room>`](https://chatie.io/wechaty/#Room.find) | Try to find a room by filter. If get many, return the first one. |
+| static | [`findAll(query: string): Promise<Room[]>`](https://chatie.io/wechaty/#Room.findAll) | Find all contacts in a room |
+| static | [`load(query: string): Room`](https://chatie.io/wechaty/#Room.load) | load room by room id |
 | property | `id: readonly string` |  |
-| event | `join` | emit when anyone join any room |
-| event | `topic` | emit when someone change room topic |
-| event | `leave` | emit when anyone leave the room |
-| event | `invite` | emit when receive a room invitation |
-| method | `sync(): <Promise<void>` | force reload data for room, sync data from lowlevel API again.
-| method | `say(text: string): Promise<void>` | Send text,media file, contact card, or text with mention @mention contact inside Room |
-| method | `add(contact: Contact): Promise<void>` | Add contact in a room |
-| method | `del(contact: Contact): Promise<void>` | Delete a contact from the room |
-| method | `quit(): Promise<void>` | Bot quit the room itself |
-| method | `topic(): Promise<string>` | GET topic from the room |
-| method | `topic(newTopic: string): Promise<void>` | SET topic from the room |
-| method | `announce(text: string): Promise<void>` | SET/GET announce from the room |
-| method | `qrcode(): Promise<string>` | Get QR Code of the Room from the room, which can be used as scan and join the room. |
-| method | `alias(contact: Contact): Promise<string>` | Return contact's roomAlias in the room |
-| method | `roomAlias(contact: Contact): Promise<string|null>` | Same as function alias|
-| method | `has(contact: Contact): Promise<boolean>` | Check if the room has member `contact` |
-| method | `memberAll(query?: string): Promise<Contact[]>` | Find all contacts or with specific name in a room |
-| method | `member(query: string): Promise<null | Contact>` | Find all contacts in a room, if get many, return the first one. |
-| method | `memberList():Promise<Contact[]>` | get all room member from the room|
-| method | `owner(): null \| Contact` | Get room's owner from the room. |
+| event | [`join`](https://chatie.io/wechaty/#Room+on) | emit when anyone join any room |
```

Comment:
Please change link to `https://docs.chatie.io/` since we have update all new docs, thanks!

Reply:
@lijiarui the document in the README should point to `chatie.io/wechaty` because it is generated directly from the source code.